packages/terra-card/src/CardPadding.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides 10px of themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides 10px of themeable padding horizontal
+   */
+  hasPaddingHorizontal: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasPaddingHorizontal: false,
+  hasPaddingVertical: false,
+};
+
+
+const CardPadding = ({
+  children,
+  hasPaddingVertical,
+  hasPaddingHorizontal,
+  ...customProps
+}) => {
+  const cardPaddingClasses = cx([
+    { 'veritcal-padding': hasPaddingVertical },
+    { 'horizontal-padding': hasPaddingHorizontal },
+    customProps.className,
+  ]);
+  return <div {...customProps} className={cardPaddingClasses}>{children}</div>;
```

Comment:
I think we support both. The reasoning is if we have a card with a vertical divider that runs through the middle adn we want standard card-padded regions on the top and the bototm, we can reuse this component to acheive that.
```jsx
<card>
<card.padding>Hello</card.padding>
<hr/>
<card.padding>More content</card.padding>
</card>
```